apps/builder/app/builder/features/style-panel/controls/font-weight/font-weight-control.tsx

```diff
@@ -49,6 +49,13 @@ const useLabels = (
   availableFontWeights: Array<FontWeightItem>,
   currentWeight: string
 ) => {
+  // support named aliases
+  if (currentWeight === "normal") {
```

Comment:
strange list, there is generally more keywords than normal and bold

Reply:
lighter and bolder but they are relative so no specific value